docs/assets/documents/governance-for-hospitalNotification.md

```diff
@@ -35,23 +35,15 @@ The flow of events and rules, when sending and receiving a HospitalNotification,
 |A Sending EcoSystem **SHALL** unwrap an Acknowledgement in a VANS-envelope|
 
```

Comment:
A Sending EcoSystem **SHALL** unwrap a received Acknowledgement in a VANS-envelope|